src/Finder/TransformedFinder.php

```diff
@@ -69,6 +69,18 @@ public function findHybrid($query, $limit = null, $options = [])
         return $this->transformer->hybridTransform($results);
     }
 
+    /**
+     * @param $query
+     * @param null|int $limit
+     * @param array    $options
+     *
+     * @return array
+     */
+    public function findRaw($query, $limit = null, $option = [])
```

Comment:
You may use type hints and return types.

Reply:
Fixed.
I do not type hint `$query` because it can be an array or an \Elastica\Query object.